.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: trailing-whitespace
+        exclude: '^resotool/resoapp/static/resoapp/bootstrap-4.3.1-dist/'
+    -   id: end-of-file-fixer
+        exclude: '^resotool/resoapp/static/resoapp/bootstrap-4.3.1-dist/'
+-   repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+    -   id: black
+        exclude: '^resotool/resoapp/migrations/'
```

Comment:
Is there a reason why we should exclude migrations from being reformatted? Since they might also be edited, they should also be formatted by `Black`.

Reply:
Sure, they can be, but my assumption was, that they generally would be static. I guess there is no harm in formatting them.

Reply:
Generally, they are static, but for some optimization stuff it can be useful to edit migrations. And since we are also reformatting all other automatically generated files, for consistency sake they should should be nicely formatted, too.

Reply:
Also I think it would be a good idea to mention the pre-commit hook in `README.md`, maybe even with a small instruction on how to install it.